package.json

```diff
@@ -135,10 +135,11 @@
         "test": "node ./node_modules/vscode/bin/test"
     },
     "dependencies": {
-        "vscode-debugprotocol": "^1.32.0",
-        "vscode-debugadapter": "^1.32.0",
         "long": "^4.0.0",
         "uuid": "^3.3.2",
+        "vsce": "^1.59.0",
```

Comment:
Can you explain why `vsce` has been added as a dependency of the extension? - As far as I'm aware, it's normally used as the developer tool for publishing extensions.